portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase

```diff
@@ -509,6 +509,66 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-129536. View the configuration of widgets on content page persist after execute Clean Up Orphaned Theme Portlet Preferences."
+	@priority = "4"
+	test ViewConfigurationSettingsOfWidgetAfterCleanUpOrphanedThemePortletPreferences {
+		property test.name.skip.portal.instance = "ContentPages#ViewConfigurationSettingsOfWidgetAfterCleanUpOrphanedThemePortletPreferences";
+
+		task ("Add a web content via JSON") {
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title");
+		}
+
+		task ("Add a Asset Publisher widget to the page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addWidget(portletName = "Asset Publisher");
+		}
+
+		task ("Change the display template to Title List") {
+			Portlet.gotoPortletOptions(
+				portletName = "Asset Publisher",
+				portletOption = "Configuration");
+
+			AssetPublisherPortlet.configureDisplayTemplatePG(displayTemplate = "Title List");
+
+			IFrame.closeFrame();
+		}
+
+		task ("View web content displayed at Display Template in Asset Publisher at edit mode") {
+			AssertVisible(
+				key_assetTitle = "Web Content Title",
+				locator1 = "AP#ASSET_ABSTRACT_EDIT");
+
+			PageEditor.clickPublish();
+		}
+
+		task ("Execute Clean Up Orphaned Theme Portlet Preferences in Server Administration") {
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "System",
+				panel = "Control Panel",
+				portlet = "Server Administration");
+
+			ServerAdministration.executeServerResourcesActions(actionsDescription = "Clean Up Orphaned Theme Portlet Preferences");
+		}
+
+		task ("View web content displayed at Display Template in Asset Publisher at view mode") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			AssertElementPresent(
+				key_assetTitle = "Web Content Title",
+				locator1 = "AP#ASSET_ABSTRACT_EDIT");
+		}
+	}
```

Comment:
Add steps to view web content displayed at Title List in Asset Publisher at edit mode